geoindicators/pom.xml

```diff
@@ -45,6 +45,18 @@
             <artifactId>h2gis-network</artifactId>
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.haifengl</groupId>
+            <artifactId>smile-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.github.haifengl</groupId>
+            <artifactId>smile-netlib</artifactId>
```

Comment:
Is this dep necessary ?

Reply:
necessary I do not think so but it seems it optimizes the matrix manipulation:
see http://haifengl.github.io/faq.html